tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -360,7 +384,7 @@ def _gen_kernel_library(
             type = type,
             output_type = output_type,
         )
-        for (type, output_type, jit) in all_kernels
+        for (type, output_type, jit, index_64bit) in all_kernels
     ] + ["//tensorflow/compiler/mlir/tools/kernel_gen:tf_framework_c_interface"]
 
```

Comment:
Should be jit_i64_indexed_for_large_tensors

Reply:
Name changed